StudyPathTraker/StudyPathTraker/Custom Views/SPRoundedButton.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  SPRoundedButton.swift
+//  StudyPathTraker
+//
+//  Created by Rafael Lopez on 6/09/18.
+//  Copyright © 2018 Jerti. All rights reserved.
+//
+
+import UIKit
+
+class SPRoundedButton: UIButton {
+    
+    let cornerSize: CGFloat = 8.0
+
+    override func draw(_ rect: CGRect) {
+        layer.cornerRadius = cornerSize
```

Comment:
make this here is a bad practice, you should write this on init with frame or init with coder. read about the UIView lifecycle.